lib/game.rb

```diff
@@ -6,18 +6,24 @@ class Game
   def initialize(input:, output:)
     @input = input
     @output = output
+    @grid = {}
   end
 
   def play
     grid_view.render
     get_player_input
+    grid_view.render
   end
 
   private
 
-  attr_reader :input, :output
+  attr_reader :grid, :input, :output
 
   def get_player_input
```

Comment:
This method name doesn't make much sense anymore. The method `player_coordinates` is really about getting input, and this is about saving the input as data.